dnf/transaction_sr.py

```diff
@@ -308,7 +308,7 @@ def _replay_pkg_action(self, pkg_data):
                 query = query_repo.union(query.installed())
 
         if not query:
-            self._raise_or_warn(self._skip_unavailable, _('Cannot find rpm nevra "{nevra}".').format(nevra=nevra))
+            self._raise_or_warn(self._skip_unavailable, _('No package %s available.') % nevra)
```

Comment:
You need to update the transaction-sr tests for this change. It doesn't fit with the other messages, so I'm still not too happy about it.

Reply:
I updated the tests and also changed the code to print different messages in transaction-sr and history.